modules/channel-web/src/views/lite/main.tsx

```diff
@@ -335,9 +352,9 @@ class Web extends React.Component<MainProps> {
     const { isEmulator, stylesheet, extraStylesheet } = this.props.config
     return (
       <React.Fragment>
+        {!isIE && <Stylesheet href={'assets/studio/ui/public/external/font-roboto.css'} />}
```

Comment:
Why the change ? We no longer load channel web font css ? I don't think the channel should be aware of the studio